x11-libs/libXaw3dXft/libXaw3dXft-1.6.2h-r1.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools xorg-3
+
+DESCRIPTION="Xaw3dXft library"
+HOMEPAGE="https://sourceforge.net/projects/sf-xpaint/"
+SRC_URI="mirror://sourceforge/sf-xpaint/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+IUSE="unicode xpm"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXft
+	x11-libs/libXmu
+	x11-libs/libXt
+	xpm? ( x11-libs/libXpm )"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+BDEPEND="
+	sys-devel/flex
+	virtual/yacc"
+
+src_prepare() {
+	default
+
+	#859877 replace version in configure.ac (e.g., 1.6.2 -> 1.6.2h)
+	if [[ ${PV} == *[a-z] ]]; then
+		local AC_PV=${PV//./\\.}
+		local AC_PV=${AC_PV:0:-1}
+		sed -i -e "s/\[${AC_PV}\]/\[${PV}\]/" configure.ac
+
+		eautoreconf
+	fi
+}
```

Here's an alternative suggestion:

```suggestion
QA_PKGCONFIG_VERSION="${PV//[!0-9.]}"
```

Allows you to remove autotools eclass inherit too.